File: day-1/day-1-2.py
with open("input.txt") as f:
	a = [int(f.readline()[:-1]) for i in range(200)]
	
for i in a:
	for j in a:
		for k in a:
			if i + j + k  == 2020:
				print(i*j*k)
			else:	
				pass

File: day-1/day-1-2.cpp
# include <fstream>
# include <iostream>
# include <vector>

using namespace std;

int main()
{
	ifstream file("input.txt");
	vector<int> v;
	while (file.eof() == false){
		int a;
		file >> a;
		v.push_back(a);
	}
	file.close();
	
	for(int a : v)
		for(int b : v)
			for(int c : v)
				if(a + b + c == 2020)
					cout << a*b*c << endl;
}
